src/coreclr/ildasm/dres.cpp

```diff
@@ -155,7 +155,7 @@ DWORD   DumpResourceToFile(_In_ __nullterminated WCHAR*   wzFileName)
     DWORD ret = 0;
     DWORD   dwResDirRVA;
     DWORD   dwResDirSize;
-    unsigned ulNumResNodes=0;
+    unsigned short ulNumResNodes=0;
```

Comment:
You might also be able to use `uint16_t` which is a bit more explicit

Reply:
Do we use 'uint16_t' commonly? 'unsigned short' is used multiple times in the body of this function.

Reply:
It depends on the code you're looking at and how new it is.

`unsigned short` doesn't have a "well-defined" size in C and just happens to currently be the same as `uint16_t` on all platforms we currently target.

`uint16_t` is always going to be 16-bits on all platforms and so its often better to be explicit where feasible (remove ambiguity, reduce potential bugs, etc).